src/Traits/LogsActivity.php

```diff
@@ -18,7 +18,7 @@ trait LogsActivity
     public function getActivitylogOptions(): LogOptions
```

Comment:
We need to delete this file. The docs now say to create your own.

Reply:
I can delete the file from the Github interface (no problem) but I didn't want to break @promatik 's tests.